application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -141,4 +141,13 @@ public static String abbreviate(String text, int maxLength) {
         return text.substring(0, maxLength - ABBREVIATION.length()) + ABBREVIATION;
     }
 
+    /**
+     * Mentions a guild channel by its id
+     *
+     * @param channelId the channel id to mention
+     * @return channel mention
+     */
+    public static String mentionChannelById(long channelId) {
+        return "<#%s>".formatted(channelId);
```

Comment:
Please add a comment that links to where this is from and why it had to be created. I.e.
```java
// Clone of JDAs Channel#getAsMention, but unfortunately channel instances can not be created out of just an ID, unlike User#fromId
```
then people also know directly that this method can be deleted once JDA adds this method for channels as well

Reply:
this should rather be a comment than part of the javadoc. javadoc is to document the method, not to give internal implementation notes.

Reply:
please move the comment inside the method, thanks

Reply:
done